crates/tinymist-query/src/goto_definition.rs

```diff
@@ -65,12 +66,28 @@ mod tests {
         snapshot_testing("goto_definition", &|world, path| {
             let source = world.source_by_path(&path).unwrap();
 
+            let docs = find_module_level_docs(&source).unwrap_or_default();
+            let properties = get_test_properties(&docs);
+            let must_compile = properties
```

Comment:
could you lift the repeated code as a function in `tests.rs`? I guess you copied it from `completion.rs`.

Reply:
sure, I made a very light abstract there